lldb/tools/lldb-dap/DAP.h

```diff
@@ -192,8 +192,6 @@ struct DAP {
   std::mutex call_mutex;
   std::map<int /* request_seq */, ResponseCallback /* reply handler */>
       inflight_reverse_requests;
-  StartDebuggingRequestHandler start_debugging_request_handler;
-  ReplModeRequestHandler repl_mode_request_handler;
```

Comment:
btw, do you use the REPL mode at meta?

Reply:
I do not remember clearly but I think we have hit issues internally because `auto` REPL mode will perform variables lookup in the background which can trigger very slow performance code path for an internal large target.